interfaces/AW-App/src/app/account/account.page.spec.ts

```diff
@@ -30,4 +42,46 @@ describe('AccountPage', () => {
   it('should create', () => {
     expect(component).toBeTruthy();
   });
+
+  it('ngOnInit: should set up variables', () => {
+    expect(component.isLoggedIn).toBeDefined();
+  });
+
+  it('doLogin: should call login of authService', () => {
+    spyOn(event, "preventDefault");
+    component.doLogin(event);
+    expect(event.preventDefault).toHaveBeenCalled();
```

Comment:
Is the `event`-object chosen as 'just an example of a parameter', or is it the suggestion to use this default browser-behaviour as a feature in our tests?

Is it to check if "is the first line of code in the method executed"? Or something else/more?

As the